### PR TITLE
Update commonobject.class.php add in setUpperOrLowerCase

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# New [*add in setUpperOrLowerCase*]
add in setUpperOrLowerCase

### CODE REVIEW

File: htdocs/core/class/commonobject.class.php [hunk 1]
```diff
@@ -638,12 +638,15 @@ public function setUpperOrLowerCase()
...
 			$this->name = dol_ucwords(dol_strtolower($this->name));
+			$this->name_alias = dol_ucwords(dol_strtolower($this->name_alias));
 		}
 		if (!empty($conf->global->MAIN_ALL_TO_UPPER)) {
 			$this->lastname = dol_strtoupper($this->lastname);
 			$this->name = dol_strtoupper($this->name);
+			$this->name_alias = dol_strtoupper($this->name_alias);
 		}
 		if (!empty($conf->global->MAIN_ALL_TOWN_TO_UPPER)) {
+			$this->address = dol_strtoupper($this->town);
```
Comment (frederic34) [inline htdocs/core/class/commonobject.class.php:649]: @bb2a town -> address ?

Comment (bb2a) [reply htdocs/core/class/commonobject.class.php:649]: You would like me to add global value for address?

Comment (frederic34) [reply htdocs/core/class/commonobject.class.php:649]: $this->address = dol_strtoupper($this->town);  ?

Comment (bb2a) [reply htdocs/core/class/commonobject.class.php:649]: Oups sorry

Comment (bb2a) [reply htdocs/core/class/commonobject.class.php]: I change

**Pull request merged**